Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -4401,31 +4401,19 @@ def test_failure(mocker):
         When:
             - Calling execute_command.
         Then:
-            - Assert that the original error is returned to War-Room (using demisto.results).
-            - Assert an error is returned to the War-Room.
-            - Function ends the run using SystemExit.
+            - Assert that DemistoException is raised with the original error.
         """
         from CommonServerPython import execute_command, EntryType
         error_entries = [
             {'Type': EntryType.ERROR, 'Contents': 'error number 1'},
-            {'Type': EntryType.NOTE, 'Contents': 'not an error'},
             {'Type': EntryType.ERROR, 'Contents': 'error number 2'},
         ]
-        demisto_execute_mock = mocker.patch.object(demisto, 'executeCommand',
-                                                   return_value=error_entries)
-        demisto_results_mock = mocker.patch.object(demisto, 'results')
+        demisto_execute_mock = mocker.patch.object(demisto, 'executeCommand', return_value=error_entries)
 
-        with raises(SystemExit, match='0'):
+        with raises(DemistoException, match='Failed to execute'):
             execute_command('bad', {'arg1': 'value'})
 
         assert demisto_execute_mock.call_count == 1
-        assert demisto_results_mock.call_count == 1
-        # first call, args (not kwargs), first argument
-        error_text = demisto_results_mock.call_args_list[0][0][0]['Contents']
-        assert 'Failed to execute bad.' in error_text
-        assert 'error number 1' in error_text
-        assert 'error number 2' in error_text
-        assert 'not an error' not in error_text
```

Comment:
for an already existing unit tests, please make sure to let them pass, not to delete them.

Reply:
`assert demisto_results_mock.call_count == 1`
This line fails, I think that's because DemistoException is being thrown on error instead of demisto.results execution